docs/development/install.rst

```diff
@@ -84,11 +85,34 @@ Check that everything works
 #. click on the "View docs" button to browse the documentation, and verify that it works
 
 
+.. admonition:: Front End Development
+
+  If you were sent to this page from the *Front End Development Getting Started* section, you can now `go back and follow the instructions there </development/front-end.html#getting-started>`_.
+
+
+
+
 Working with Docker Compose
 ---------------------------
 
-We wrote a wrapper with ``invoke`` around ``docker-compose`` to have some shortcuts and
-save some work while typing docker compose commands. This section explains these ``invoke`` commands:
+After the initial setup you'll need to run a few commands each time.
+
+#. get the containers running:
+
+  .. prompt:: bash
+
+    inv docker.up  # Options: --no-search --no-reload
+
+#. do stuff
+
+
+List of available commands
+""""""""""""""""""""""""""
+
+.. admonition:: Note
+
+  We wrote a wrapper with ``invoke`` around ``docker-compose`` to have some shortcuts and
+  save some work while typing docker compose commands.
```

Comment:
I would keep the same title we have here and add an extra paragraph mentioning that this is the extensive list of all available commands (like the manpage for our tool) which is not required to know it completely and there may be commands that the developer will never use. Communicating that this has to be seen as a reference more than something that the developer must know.

Reply:
I made some changes here. Kept the title and added the paragraph:

> Bellow is the list of all available commands.

Also tried to work in a more extensive paragraph with your suggestions but I think that's mostly implicit by having the previous section and that extending on it here would add unnecessary density.

Let me know what you think of the changes.